pkgs/by-name/er/ersatztv/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  fetchFromGitHub,
+  dotnetCorePackages,
+  buildDotnetModule,
+  sqlite,
+  ffmpeg,
+  which,
+}:
+
+buildDotnetModule rec {
+  pname = "ersatztv";
+  version = "0.8.7-beta";
+
+  src = fetchFromGitHub {
+    owner = "ErsatzTV";
+    repo = "ErsatzTV";
+    rev = "v${version}";
+    sha256 = "sha256-G3yw9aYZABEyghCddpAoO71wrjVbWiVJALEMZSbr2Lg=";
+  };
+
+  propagatedBuildInputs = [ sqlite ];
+
+  buildInputs = [ ffmpeg ];
+
+  projectFile = "ErsatzTV/ErsatzTV.csproj";
+  executables = [
+    "ErsatzTV"
+    "ErsatzTV.Scanner"
+  ];
+  nugetDeps = ./nuget-deps.nix;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  # ETV uses `which` to find `ffmpeg` and `ffprobe`
```

Comment:
Can we hardcode that instead?

Reply:
Unfortunately not. The only ways I could find to set it are letting it autodiscover the FFmpeg path, or setting it manually in the GUI.